examples/custom.py

```diff
@@ -45,17 +45,17 @@ class MyAstroidChecker(BaseChecker):
         ),
     )
 
-    def visit_call(self, node):
-        """Called when a :class:`.astroid.node_classes.Call` node is visited.
+    def visit_call(self, node: nodes.Call) -> None:
+        """Called when a :class:`.nodes.Call` node is visited.
 
         See :mod:`astroid` for the description of available nodes.
 
         :param node: The node to check.
-        :type node: astroid.node_classes.Call
+        :type node: nodes.Call
```

Comment:
I'm wondering if we should keep the docstring type when the type is in the signature now.

Reply:
I think removing would be fine.

However, I'm wondering if the typing shouldn't be `astroid.nodes.Call`. Not sure what is standard but I always type with whatever `type()` would return. Imo, this makes it easier for new contributors to understand what is going on without remembering all imports and aliases.

Reply:
I think it's good to remove it from the docstring. Type annotations are usually more expressive than what you can do in a docstring. Plus there's the potential for the two places to go out of sync. Annotations show up in `help()` and in documentation (and in documentation it can even be resolved to a fully qualified name).